va/va.h

```diff
@@ -1439,6 +1454,33 @@ typedef struct _VAEncMiscParameterBufferQualityLevel {
 } VAEncMiscParameterBufferQualityLevel;
 
 /**
+ * \brief Quantization settings for encoding.
+ *
+ * Some encoders support special types of quantization such as trellis, and this structure
+ * can be used by the app to control these special types of quantization by the encoder.
+ */
+typedef struct _VAEncMiscParameterQuantization
+{
+    union
+    {
+    /* if no flags is set then quantization is determined by the driver */
+        struct
+        {
+	    /* \brief disable trellis for all frames/fields */
+            unsigned int disable_trellis : 1;
+	    /* \brief enable trellis for I frames/fields */
+            unsigned int enable_trellis_I : 1;
+	    /* \brief enable trellis for P frames/fields */
+            unsigned int enable_trellis_P : 1;
+	    /* \brief enable trellis for B frames/fields */
+            unsigned int enable_trellis_B : 1;
```

Comment:
Could yo change the type of value to uint64_t so that we have more reserved bits for future extension? I will change existent data structures as well.